style.css

```diff
@@ -0,0 +1,352 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    max-width: 600px;
+    margin: 0 auto;
+    background-color: #D4CDAB;
+    font-family: 'Dosis', sans-serif; 
+}
+
+img {
+    max-width: 100%;
+}
+
+/* ******Header******** */
+.header {
+    display: flex;
+    /* text-decoration: column; */
+    justify-content: space-between;
+    background-color: #97A7B3;
+    /* height: 150px; */
+}
+
+.header-desk {
+    display:none;
+}
+
+.header-sub {
+    font-family: 'Eczar';
```

Comment:
You should always put a fallback font as the last one (like `serif`, `sans-serif`, etc).